### ISSUE TITLE
Installation on HA 2022.06

### ISSUE THREAD

**Issue by jmsucasas** [kvj/hass_nuki_ng#68]
Hi,
Is not possible to install this pluggin on HA 2022.06.  If I try to install it as an add-on,  HA says that is not possible and the option to install it as a custom integration does not exist.  
Thanks in advance

**Comment by jmcollin78** [comment]
Same case, but was solved by opening a Web Browser in private navigation mode. A matter of browser cache certainly

**Comment by alexdelprete** [comment]
> Hi, Is not possible to install this pluggin on HA 2022.06. If I try to install it as an add-on, HA says that is not possible and the option to install it as a custom integration does not exist. Thanks in advance

Did you read the instructions on the repo? You first install the custom component via HACS, then you configure the integration.

![image](https://user-images.githubusercontent.com/7027842/175719636-f1359232-e2cc-40cd-85e6-1f0675063914.png)

**Comment by alexdelprete** [comment]
> Same case, but was solved by opening a Web Browser in private navigation mode. A matter of browser cache certainly

The HACS custom component installation didn't work for you or are you referring to the configuration of the integration? If the integration does not appear on the list, it's a known HA issue, you simply do a hard-refresh of the page in the browser (ctrl-F5), then you add it again and you should see it in the list (name Nuki Lock, white logo).

Doing these configs with private sessions could lead to weird issues because probably HA needs to use temp cookies or session variables etc.

**Comment by jmcollin78** [comment]
@alexdelprete  yes exactly. Opening a new private session also works but you have to login again.
Thank's for the tips.
